charactersheet/charactersheet/models/common/weapon.js

```diff
@@ -136,6 +136,27 @@ function Weapon() {
         }
     });
 
+    self.weaponRangeLabel = ko.pureComputed(function() {
+        if (self.weaponType().toLowerCase() === 'ranged'){
```

Comment:
My only response is that we know the data that is going in, and it's, currently, impossible to have any input that's not _ranged_ or _melee_.